### PR TITLE
chore: set 1000 as default max limit

### PR DESCRIPTION
**Pull request by BonapartePC**
# Description

set 1000 as default max limit for adaptive rate limiting

## Security

- [x] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

**Review by cisse21 — APPROVED** [review]

File: router/throttler/adaptive.go [hunk 1]
```diff
@@ -16,6 +16,8 @@ type adaptiveThrottleConfig struct {
 	maxLimit func() int64
 }
 
+const DefaultMaxLimit = 1000 // 1000 requests per second
```
Comment (achettyiitr) [inline router/throttler/adaptive.go:19]: Does this need to be exported? It's being used within the package only?

File: router/throttler/factory_test.go [hunk 2]
```diff
@@ -103,6 +103,16 @@ func TestFactory(t *testing.T) {
...
 	})
+
+	t.Run("when no limit is set", func(t *testing.T) {
+		config := config.New()
+		config.Set("Router.throttler.adaptive.enabled", true)
+		f, err := NewFactory(config, nil)
+		require.NoError(t, err)
+		defer f.Shutdown()
+		ta := f.Get("destName", "destID")
+		require.Equal(t, int64(1000), ta.getLimit())
```
Comment (achettyiitr) [inline router/throttler/factory_test.go:114]:
```suggestion
		require.EqualValues(t, DefaultMaxLimit, ta.getLimit())
```

File: router/throttler/factory_test.go [hunk 3]
```diff
@@ -103,6 +103,16 @@ func TestFactory(t *testing.T) {
 			"destType":      "destName",
 		}).LastValue())
 	})
+
+	t.Run("when no limit is set", func(t *testing.T) {
+		config := config.New()
+		config.Set("Router.throttler.adaptive.enabled", true)
```
Comment (achettyiitr) [inline router/throttler/factory_test.go:109]: [Optional]: Variable 'config' collides with imported package name 
```suggestion
		conf := config.New()
		conf.Set("Router.throttler.adaptive.enabled", true)
```

**Review by achettyiitr — APPROVED** [review]

File: router/throttler/adaptive.go (hunk 1 above)
Comment (BonapartePC) [reply router/throttler/adaptive.go]: Made the change

File: router/throttler/factory_test.go (hunk 2 above)
Comment (BonapartePC) [reply router/throttler/factory_test.go]: Done

**Pull request merged**